unicodetools/src/main/java/org/unicode/picker/CharData.java

```diff
@@ -50,8 +50,8 @@
 {"–","EN DASH"},
 {"—","EM DASH"},
 {"―","HORIZONTAL BAR"},
-{" ","LINE SEPARATOR"},
-{" ","PARAGRAPH SEPARATOR"},
+{"","LINE SEPARATOR"},
+{"","PARAGRAPH SEPARATOR"},
```

Comment:
```java
{"\u2028","LINE SEPARATOR"},
{"\u2029","PARAGRAPH SEPARATOR"},
```
(apparently the literal characters got lost; we should put them back using escapes)

Reply:
good catch, will fix.

I did _not_ cause the odd lines later in that file:

```java
  public static String[][][] CATEGORIES = {
{{"Symbol"},
/*116,6*/ {"Alchemical Symbols@Other ","A2j1dA"},
```